lightning_server.cc

```diff
@@ -15,7 +15,15 @@ LightningServer::LightningServer(const char* file_name)
   config_parser_.Parse(file_name, &config_);
   std::cout << config_.ToString() << std::endl;
   ServerConfig ConfigWrapper(config_);
```

Comment:
To follow our previous casing conventions, we'd want to make this `config_wrapper`